### DISCUSSION TITLE
How do export fastdds all qos configs writed in code to xml profiles?

### DISCUSSION THREAD

**Discussion by chunyisong** [eProsima/Fast-DDS#3305] [Q&A]
I write Fastdds and write qos in code.Now I want to export all the configs to profiles in a xml.
I want to know if there has a simple way.Thanks.

**Comment by JLBuenoLopez** [comment]
Hi @chunyisong 

I am sorry to say that Fast DDS does not have a feature that provides this functionality right now. :disappointed:
